tools/Uno.Sdk.Updater/Uno.Sdk.Updater.csproj

```diff
@@ -20,7 +21,8 @@
 	<Target Name="GenerateVersion" BeforeTargets="BeforeCompile">
 
 		<PropertyGroup Condition="'$(Configuration)'=='Debug'">
-			<GitVersion_FullSemVer Condition="'$(GitVersion_FullSemVer)'==''">255.255.255.255</GitVersion_FullSemVer>
+			<!-- When debugging, change this version to the next pre-release -->
+			<GitVersion_FullSemVer Condition="'$(GitVersion_FullSemVer)'==''">5.3-dev</GitVersion_FullSemVer>
```

Comment:
I'd rather not make this change. It shouldn't need to be updated over time and the 255.255.255.255 should be a good indicator of how to fix for debugging